fineract-accounting/src/main/java/org/apache/fineract/accounting/producttoaccountmapping/service/ProductToGLAccountMappingHelper.java

```diff
@@ -100,6 +100,8 @@ public void mergeProductToAccountMappingChanges(final JsonElement element, final
                 optionalProductToGLAccountMappingEntries.add("incomeFromGoodwillCreditInterestAccountId");
                 optionalProductToGLAccountMappingEntries.add("incomeFromGoodwillCreditFeesAccountId");
                 optionalProductToGLAccountMappingEntries.add("incomeFromGoodwillCreditPenaltyAccountId");
+                optionalProductToGLAccountMappingEntries.add("deferredIncomeLiabilityAccountId");
+                optionalProductToGLAccountMappingEntries.add("incomeFromDeferredIncomeAccountId");
```

Comment:
The mapping name is "INCOME FROM CAPITALIZATION"

Reply:
Renamed it.